src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -107,7 +107,7 @@ public function __construct(array $options = array(), $handler = null, MetadataB
         }
 
         $options += array(
-            'cache_limiter' => 'private_no_expire',
+            'cache_limiter' => '',
```

Comment:
the phpdoc with the standard options is wrong now

Reply:
fixed in https://github.com/symfony/symfony/commit/cf4eb4602ace6c32523133b09feab79a53f8fa2d